closure/goog/i18n/numberformat_test.js

```diff
@@ -273,6 +273,161 @@ function testGrouping() {
   fmt = new goog.i18n.NumberFormat('#');
   str = fmt.format(1234567890);
   assertEquals('1234567890', str);
+
+}
+
+function testIndiaNumberGrouping() {
+  // Test for a known grouping used and recognized in India
+  var fmt = new goog.i18n.NumberFormat('#,##,###');
+  var str = fmt.format(1);
+  assertEquals('1', str);
+  str = fmt.format(12);
+  assertEquals('12', str);
+  str = fmt.format(123);
+  assertEquals('123', str);
+  str = fmt.format(1234);
+  assertEquals('1,234', str);
+  str = fmt.format(12345);
+  assertEquals('12,345', str);
+  str = fmt.format(123456);
+  assertEquals('1,23,456', str);
+  str = fmt.format(1234567);
+  assertEquals('12,34,567', str);
+  str = fmt.format(12345678);
+  assertEquals('1,23,45,678', str);
+  str = fmt.format(123456789);
+  assertEquals('12,34,56,789', str);
+  str = fmt.format(1234567890);
+  assertEquals('1,23,45,67,890', str);
+  str = fmt.format(0);
+  assertEquals('0', str);
+  str = fmt.format(-1);
+  assertEquals('-1', str);
+  str = fmt.format(-12);
+  assertEquals('-12', str);
+  str = fmt.format(-123);
+  assertEquals('-123', str);
+  str = fmt.format(-1234);
+  assertEquals('-1,234', str);
+  str = fmt.format(-12345);
+  assertEquals('-12,345', str);
+  str = fmt.format(-123456);
+  assertEquals('-1,23,456', str);
+  str = fmt.format(-1234567);
+  assertEquals('-12,34,567', str);
+  str = fmt.format(-12345678);
+  assertEquals('-1,23,45,678', str);
+  str = fmt.format(-123456789);
+  assertEquals('-12,34,56,789', str);
+  str = fmt.format(-1234567890);
+  assertEquals('-1,23,45,67,890', str);
```

Comment:
Are all of these test cases testing something unique? If not, there's no real point in having them.